pmd-plsql/src/test/java/net/sourceforge/pmd/lang/plsql/ast/ParsingExclusionTest.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.plsql.ast;
+
+import org.junit.Test;
+
+import net.sourceforge.pmd.lang.plsql.AbstractPLSQLParserTst;
+
+public class ParsingExclusionTest extends AbstractPLSQLParserTst {
+
+    @Test
+    public void parseParsingExclusion() {
+        plsql.parseResource("ParsingExclusion.sql");
```

Comment:
I think, we should somehow verify that the AST doesn't contain the excluded elements. In the future, we might improve the parser, so that it can parse scalar subqueries in where conditions....

maybe it's easier to just write really invalid SQL that will be skipped by PMD to verify, it is indeed being skipped....